File: Data Structures & Algorithms/maximum-distance-in-arrays/submission-0.py
class Solution:
    def maxDistance(self, arrays: List[List[int]]) -> int:
        res = 0
        n = len(arrays)
        for i in range(n - 1):
            for j in range(i + 1, n):
                arr1 = arrays[i]
                arr2 = arrays[j]
                res = max(res, abs(arr1[0] - arr2[-1]))
                res = max(res, abs(arr2[0] - arr1[-1]))
        return res

File: Data Structures & Algorithms/replace-elements-with-greatest-element-on-right-side/submission-0.cpp
class Solution {
public:
    vector<int> replaceElements(vector<int>& arr) {
        vector<int> res(arr.size());
        for(int i = 0; i < arr.size(); i++) {
            int rMax = -1;
            for(int j = i + 1; j < arr.size(); j++) {
                rMax = max(rMax, arr[j]);
            }
            res[i] = rMax;
        }
        return res;
    }
};